src/wallet.cpp

```diff
@@ -118,7 +118,6 @@ bool CWallet::Unlock(const SecureString& strWalletPassphrase)
 
 bool CWallet::ChangeWalletPassphrase(const SecureString& strOldWalletPassphrase, const SecureString& strNewWalletPassphrase)
 {
-    bool fWasLocked = IsLocked();
```

Comment:
Although `fWasLocked` is not used, I wonder if this may cause regression. Have a look at `IsLocked` https://github.com/vergecurrency/VERGE/blob/8d1bd624c47f3c1643b137d2579d6766e8eae2ff/src/keystore.h#L133-L143

Line 139, holds a LOCK if it is crypted...wondering if that may have some other effect somewhere

Reply:
From my understanding is that the cs_KeyStore (CCrticialSection) object allows for thread-safe use. i.e, one thread at the time can access/modify in this case vMasterKey.  

So it should be safe to remove the unused isLocked() call. However, would be great to have this confirmed by others as well.